src/mlpack/bindings/cli/CMakeLists.txt

```
@@ -50,11 +50,20 @@ if (BUILD_CLI_EXECUTABLES)
   add_executable(mlpack_${name}
     ${name}_main.cpp
   )
+
+  if(NOT BUILD_SHARED_LIBS)
+    target_link_libraries(mlpack_${name} -static
+     mlpack
+     ${ARMADILLO_LIBRARIES}
+     ${COMPILER_SUPPORT_LIBRARIES}
+   )
+  else()
   target_link_libraries(mlpack_${name}
     mlpack
     ${ARMADILLO_LIBRARIES}
     ${COMPILER_SUPPORT_LIBRARIES}
   )
```

```suggestion
    target_link_libraries(mlpack_${name}
      mlpack
      ${ARMADILLO_LIBRARIES}
      ${COMPILER_SUPPORT_LIBRARIES}
    )
```
Forgotten indent. :)